### ISSUE TITLE
any plan for onnx to support tensorflow control flow 

### ISSUE THREAD

**Issue by yongwww** [onnx/onnx#1782]
A lot of TF models rely on control flow ops, almost all the TF object detection models like mask-rcnn. Is there any plan for onnx to support tf control flow? the related ops include Enter, Exit, Merge, Switch, etc.

**Comment by yongwww** [comment]
any update? @ebarsoum @prasanthpul

**Comment by guschmue** [comment]
I think starting with opset 8 onnx does have sufficient control flow support. We make use of this here:
https://github.com/onnx/tensorflow-onnx
Most of tensorflow control flow should be supported in tensorflow-onnx, dynamic rnn is supported as well.
Problem might be runtime support - we use 
https://github.com/microsoft/onnxruntime
which implements most of the control flow in opset 8.

**Comment by yongwww** [comment]
@guschmue Could you point me to a sample like SSD/mask-rcnn which uses control flow? I saw it was commented in the code. https://github.com/onnx/tensorflow-onnx/blob/master/tests/run_pretrained_models.yaml#L252

**Comment by guschmue** [comment]
I belief ssd/mask rcnn might still not work for other reasons. But one of my team mates is working on getting them to work. Let me check on it.

**Comment by ebarsoum** [comment]
Yes, some of the OPs for RCNN aren't in yet, we are working on it.

**Comment by prasanthpul** [comment]
control flow is supported in the spec and the ONNX Runtime. MaskRCNN from PyTorch is supported. MaskRCNN from TF looks like it may require some additional ops
